scripts/install.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+cd `dirname $0`/..
+
+if [[ "$(node --version)" < "v5" ]]; then
```

Comment:
Node 4 is a valid node. Well, there is a new change that made v4 unusable but we could fix that.